### PR TITLE
Add lines-between-class-members rule, package upgrade

### PR DESCRIPTION
**Pull request by markysoft**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+0.14.0 / 2018-03-15
```
Comment (st3v3nhunt) [inline CHANGELOG.md]: Can the date be today please.

File: .eslintrc.js [hunk 2]
```diff
@@ -26,6 +26,10 @@ module.exports = {
         devDependencies: true,
       },
     ],
+    "lines-between-class-members": [
```
Comment (st3v3nhunt) [inline .eslintrc.js]: Not sure why this hasn't been picked up by the linter. Can the double quotes become single quotes please? And if the linter was to fail if this happened against all the better.

Comment (markysoft) [reply .eslintrc.js]: I'll change it to single quotes for consistency, but single quotes in JSON is not the standard format, according to the JSON spec:

A value can be a string in double quotes, or a number, or true or false or null, or an object or an array. These structures can be nested

http://www.json.org/

Comment (markysoft) [reply .eslintrc.js]: The yarn.lock file for this commit includes the mocha upgrade, I'll close the other PR

Comment (st3v3nhunt) [reply .eslintrc.js]: Regarding the lock file it looks like it is already on `mocha@5+`. The `package.json` will need updating to keep greenkeeper happy.

**Pull request merged**